biz/auth.manager.js

```diff
@@ -19,7 +19,73 @@ class AuthManager extends BaseManager {
     this._authRepository = new AuthRepository();
   }
 
```

Comment:
this is same as user signup and user login just the validation schema is different try to use same function by passing parameters according to user and merchant!!!!